### ISSUE TITLE
Pretrained Vimeo90K model for reproducibility

### ISSUE THREAD

**Issue by ambitionforcomputervision** [hzwer/arXiv2020-RIFE#138]
Thank you for sharing your code to the community.

I was wondering whether you could share the pretrained weights for the model trained on the Vimeo90K dataset.

**Comment by hzwer** [comment]
Please see our README, we have released it.

**Comment by ambitionforcomputervision** [comment]
Oh, I see the RIFE model in the paper is the Vimeo90K model. Thank you.

Also, have you tried a warping loss instead of optical flow loss because there is some code with warping the ground truth optical flow?

**Comment by hzwer** [comment]
I'm not understand the warping loss you mentioned. You mean (warp(I0, flowt0) - gt).abs() ?
